packages/packages.json

```diff
@@ -2,38 +2,38 @@
+        "skill/valory/register_termination_abci/0.1.0": "bafybeih2phm7q2btqgqpvcaoknpveb5i7hhhkk2dcpculc6pkklelb657q",
+        "skill/valory/simple_abci/0.1.0": "bafybeihpv2hr3jpcqph25hptff5t6o5pg27ghncnlemmvje377evqosgrq",
+        "skill/valory/test_abci/0.1.0": "bafybeieiidsiuc7b5vm5i32jnxziawipcnlxjbshkzl74ugxfmxvr623bi",
         "agent/valory/abstract_abci/0.1.0": "bafybeigkxa5dxmjqrfhrpxewlgquphqpxlonwknpgcuxdrpva2gaen5g7i",
         "agent/valory/counter/0.1.0": "bafybeifh4v67rt23jh5uyqajqvc7tzxsy7utelf7arux6zhphnv6hjynza",
         "agent/valory/counter_client/0.1.0": "bafybeibyxuqo4itomksd6wvr3loblr2ba4jxa4x3wvtgr3rofpl5xueaaa",
-        "agent/valory/hello_world/0.1.0": "bafybeif6jh5ay7g4ebqz4kecfej7pj5nyv5redql2wludizbvq2m3n3vdu",
-        "agent/valory/register_reset/0.1.0": "bafybeibgkbm6tfl3tekutryclfl6gjm7mjgiyl7cudsskve24kim3eyqau",
-        "agent/valory/register_termination/0.1.0": "bafybeidnkexhvyvslx356ykpqsv5553n2dyl35d3aidvkxedicbg4pn574",
-        "agent/valory/registration_start_up/0.1.0": "bafybeigsltq5nma3nw2vfoyh4skaia6fimusy573usuyyupcchzimhxqhm",
-        "agent/valory/simple_abci/0.1.0": "bafybeihz7kihsuzip65x2pmapdqgterulierjhnsghumkhb4z4y6q7rdo4",
-        "agent/valory/test_abci/0.1.0": "bafybeihpbwntkwt47tmgzt7iewzpy2zl4ldir5oapo5ouf72ifisagc2cu",
+        "agent/valory/hello_world/0.1.0": "bafybeicumodkoobasitojpaqrrpd2nbf3gufocwnwvtc4dvks4cppyioom",
+        "agent/valory/register_reset/0.1.0": "bafybeic7rn3mpc4j3mf3lvxijrsbs6ynrqyf5eqvmm65f4byseouufqyaq",
+        "agent/valory/register_termination/0.1.0": "bafybeido2bk7ci34gteoqiz36v2q53u3dgqpixtiyldw26ic76wmhv3obe",
+        "agent/valory/registration_start_up/0.1.0": "bafybeieyz276wfx7unwmmewqu7ffzzzzrrg62wf5fd2mxbrqytxnu2n5t4",
+        "agent/valory/simple_abci/0.1.0": "bafybeieaprzxnxkx54mttugl5h4yvwulhdikbcfyvkoi3ncwgu7p45gxmm",
+        "agent/valory/test_abci/0.1.0": "bafybeifgj72n6j7kgnyltwqdquul4vucfxckayfdwwka6dtrws454nu7ua",
         "service/valory/counter/0.1.0": "bafybeigfid3zs3mctvxy7ztxbndz2is542oxcz2hznh4lfvtqimov4dhlu",
-        "service/valory/hello_world/0.1.0": "bafybeica6d3y6mb7lgv7wvr6dj2yidflmevd7opu3rjc3hgdq4ni7nqf2i",
-        "service/valory/register_reset/0.1.0": "bafybeib7gdxx76dv2hszzclmyamff2qjjgr3smdyp6islsvnlpwtb6wogi",
-        "service/valory/simple_abci/0.1.0": "bafybeihrbix7zrbv4yvjjdcpmzsupdov5ffgoq7a2ld3uj3gpb5nj6hl24",
-        "skill/valory/safe_deployment_abci/0.1.0": "bafybeiedso32a73rq37y24jaaiezvrzic4dmt4zghfy5pqzdwkhaw2rzzu"
+        "service/valory/hello_world/0.1.0": "bafybeie5dicy4addou6fhiiag7scueieem55dux62vdjrybo2b5nosmuze",
+        "service/valory/register_reset/0.1.0": "bafybeifgm3blejm257kx2kjarjridhgnrolo7skxakfuihcxw5bjy54xri",
+        "service/valory/simple_abci/0.1.0": "bafybeianli7tnsxsui67tzubger5bbwk24i5kuvg7arxs4fdqtyghhy3l4",
+        "skill/valory/safe_deployment_abci/0.1.0": "bafybeigsjva3vdywnzbwnwvbbvuzx5che4xagtkldkvux2t4bzr6mjjyma"
     },
     "third_party": {
```

Comment:
Did none of them change during the last release?

Reply:
They did https://github.com/valory-xyz/open-autonomy/pull/1534/files#diff-e459cec4c3e84b7f458724acf6256afb5eb4d16d0fe4f9ff97db9a57e95e8cbbR2